src/test/java/sirius/web/http/WebServerSpec.groovy

```diff
@@ -277,70 +277,6 @@ class WebServerSpec extends BaseSpecification {
         '{"success":true,"error":false,"test":true}' == data
     }
 
-    /**
```

Comment:
Are these tests redundant? Will they return?

Reply:
Sorry, did not intend to delete them